Admin/PageAdmin.php

```diff
@@ -45,6 +51,7 @@ protected function configureListFields(ListMapper $listMapper)
     protected function configureFormFields(FormMapper $formMapper)
     {
         parent::configureFormFields($formMapper);
+        $ivoryCKeditorAvailable = class_exists('Ivory\CKEditorBundle\IvoryCKEditorBundle') && $this->ivoryCkeditor;
```

Comment:
we should not thrown an exception: we just fallback to textarea if the bundle is not available.

I would like to have this even more configurable (by allowing to configure field type and field options, both with a default based on the available bundles). But I'm not sure if we're over doing the configuration side for a bundle in such cases.

In 2.0, we want to extra all form stuff from the admin (so we can use it for Kunstmaan as well). We can make such things customizable by extending the form type in such cases.